pom.xml

```diff
@@ -12,13 +12,13 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.2.6.RELEASE</version>
-        <relativePath/> <!-- lookup parent from repository -->
+        <version>2.3.0.RELEASE</version>
+        <relativePath/> <!-- lookup parent from documents -->
```

Comment:
What does this do?

Reply:
Latest version

Reply:
<relativePath/> means latest version?

Reply:
Oh no, I thought you were asking why did we move to 2.3.0

Reply:
Yeah sorry when I put in `<relativePath />` without the back ticks, github hid the relativePath thinking it was an html tag.

So what exactly is relativePath for?